src/components/Verify.js

```diff
@@ -857,7 +866,7 @@ function SidePanel(props) {
                   onClick={() => setRejectionReason('unapproved_tree')}
                   value="unapproved_tree"
                   control={<Radio />}
-                  label="Not an approved tree"
+                  label="Not an approved capture"
```

Comment:
My interpretation of this rejection reason is that the type of tree being captured is not approved for inclusion, for example an invasive species.